apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue

```diff
@@ -86,28 +85,30 @@ const hoursMatrix: string[] = range(24).map((hour) => hour.toString());
 const selectedUtcHoursSet = new Set<string>();
 const state = reactive({
     timezone: computed<string>(() => store.state.user.timezone),
-    isAutoSchedule: true,
-    isAllHoursSelected: computed<boolean>(() => state.selectedUtcHours.length === size(hoursMatrix)),
-    selectedUtcHours: [] as string[],
+    isAllHoursSelected: computed<boolean>(() => collectorFormState.scheduleHours.length === size(hoursMatrix)),
     timezoneAppliedHours: computed<string[]>(() => {
-        if (state.timezone === 'UTC') return state.selectedUtcHours;
+        if (state.timezone === 'UTC') return collectorFormState.scheduleHours;
         // set an hour as utc and get the hour in timezone
-        return state.selectedUtcHours.map((utcHour) => dayjs.utc()
+        return collectorFormState.scheduleHours.map((utcHour) => dayjs.utc()
             .hour(Number(utcHour)).tz(state.timezone)
             .get('hour')
             .toString()).sort((a, b) => Number(a) - Number(b));
     }),
     loading: computed<boolean>(() => collectorFormState.originCollector === null),
 });
 
-const updateSelectedHoursAndEmit = () => {
+const updateSelectedHours = () => {
     const hours: string[] = Array.from(selectedUtcHoursSet.values());
-    state.selectedUtcHours = hours;
-    emits('update:hours', hours);
+    collectorFormStore.$patch({
+        scheduleHours: hours,
+    });
 };
 
-const handleChangeToggle = () => {
-    // TODO: change state of toggle button
+const handleChangeToggle = (value: boolean) => {
+    collectorFormStore.$patch({
```

Comment:
praise: 😎$patch😎

Reply:
But.. there's no log left in the devtool..